src/sbt-test/sbt-header/simple/build.sbt

```diff
@@ -0,0 +1,21 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
```

Comment:
I've tried to get this working, but I don't understand how to access `scalaSource` when I simply use `scalaSource.value` inside the task I get 

```
[info] [error] References to undefined settings:
[info] [error]
[info] [error]   *:scalaSource from *:checkFileContents (/private/var/folders/66/6_p15z9j15b6gmgyc19g42cw0000gn/T/sbt_5b6a73b1/simple/build.sbt:8)
[info] [error]      Did you mean compile:scalaSource ?
```

using `val base = (scalaSource in Compile).value` gives me a weird `ClassNotFoundException`.

Also what do you mean by outdated operators? I'm not using any deprecated operators as far as I can see.